src/python/WMCore/WMBS/CreateWMBSBase.py

```diff
@@ -278,15 +278,15 @@ def __init__(self, logger = None, dbi = None, params = None):
           """CREATE TABLE wmbs_job (
              id           INTEGER       PRIMARY KEY AUTO_INCREMENT,
              jobgroup     INTEGER       NOT NULL,
-             name         VARCHAR(255),
+             name         VARCHAR(70),
              state        INTEGER       NOT NULL,
              state_time   INTEGER       NOT NULL,
              retry_count  INTEGER       DEFAULT 0,
              couch_record VARCHAR(255),
              location     INTEGER,
              outcome      INTEGER       DEFAULT 0,
-             cache_dir    VARCHAR(767)  DEFAULT 'None',
-             fwjr_path    VARCHAR(767),
+             cache_dir    VARCHAR(70)  DEFAULT 'None',
+             fwjr_path    VARCHAR(70),
```

Comment:
Alan,
maria DB does not support fields more than 255, and that was the reason for
change. Otherwise I can't proceed with changes.

For unit tests there is no need to insert gigantic names. If we can't be
flexible we'll not move forward with anything.

V.

On  0, Alan Malta Rodrigues <notifications@github.com> wrote:
> >               state        INTEGER       NOT NULL,
> >               state_time   INTEGER       NOT NULL,
> >               retry_count  INTEGER       DEFAULT 0,
> >               couch_record VARCHAR(255),
> >               location     INTEGER,
> >               outcome      INTEGER       DEFAULT 0,
> > -             cache_dir    VARCHAR(767)  DEFAULT 'None',
> > -             fwjr_path    VARCHAR(767),
> > +             cache_dir    VARCHAR(70)  DEFAULT 'None',
> > +             fwjr_path    VARCHAR(70),
> 
> I believe you mistakenly changed the length of these columns... it's a big NO for this change, we store the whole path here and even 767 chars is not enough for TaskChain.
> The same comment probably applies for other columns in other tables that you changed.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/dmwm/WMCore/pull/6260/files#r39894075